capstone/src/main/java/com/google/univiz/scorecard/ScorecardData.java

```diff
@@ -0,0 +1,76 @@
+package com.google.univiz.scorecard;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+
+/**
+ * ScorecardData is a class representing a single university/college/institution from the parsed
+ * JSON data directly from the CollegeScorecard API. The parameters come directly from the
+ * CollegeScorecard API Data Dictionary
+ * (https://collegescorecard.ed.gov/assets/CollegeScorecardDataDictionary.xlsx), which outlines the
+ * type and description of all available data elements. The parameters used here are a select few
+ * data elements the API provides.
+ */
+@GenerateTypeAdapter
+@AutoValue
+abstract class ScorecardData {
+
+  /** @return The human-readable name for the school/college/university. */
```

Comment:
The first line of a method javadoc should be a summary of its behavior. If describing the return object is the summary explanation then don't skip the summary in favor of the @return section. Just say "Returns the human-readable..." and then skip the @return. You only need the @return line if you want to further clarify details about the returned object. Like for example whether the method returns null or whether it returns an empty string or not.

Take a look at [javadoc.pdf page 5](http://dr/corp/drive/folders/14pg9M6eoAC_yM1XjC2IuocxX80R0lMXu).